replica/table.cc

```diff
@@ -1470,7 +1470,7 @@ table::sstables_as_snapshot_source() {
             return make_compacting_reader(
                 std::move(reader),
                 gc_clock::now(),
-                [](const dht::decorated_key&) { return api::min_timestamp; },
+                [](const dht::decorated_key&) { return api::max_timestamp; },
                 _compaction_manager.get_tombstone_gc_state(),
```

Comment:
I'm confused that we have both this function and tombstone_gc_state. @denesb aren't they redundant? Which one will apply?

Reply:
Ah, I guess the second one is used to check if a tombstone is eligible for purging, and the first one is used to check an eligible tombstone should be held (instead of purged) because it is deleting data in other sstables. The cache doesn't care about other sstables.

So the patch looks correct, if my understanding is right.

Reply:
That is my understanding as well.

Reply:
ok, I was just rusty.